operators/src/adapters/raster_time.rs

```diff
@@ -188,15 +191,15 @@ mod tests {
                 data: vec![
                     RasterTile2D {
                         time: TimeInterval::new_unchecked(0, 5),
-                        tile_position: [0, 0].into(),
+                        tile_position: [-1, 0].into(),
```

Comment:
What was the rationale behind changing the method and the test? Was the test previously wrong?

Reply:
as discussed the Mockrastersources previously didn't filter spatially and now it does. It then became apparent that the test data are wrong (tile index doesn't correspond to coordinates) which went unnoticed because the mockrastersource always returned the same tile ignoring the query bbox.